bsp/m5stack_core/idf_component.yml

```
@@ -0,0 +1,22 @@
+version: "2.7.0"
```

```suggestion
version: "1.0.0"
```
This is version of the BSP